pyproject.toml

```diff
@@ -17,6 +17,9 @@ Documentation = "https://github.com/CCRI-POPROX/poprox-recommender#readme"
 Issues = "https://github.com/CCRI-POPROX/poprox-recommender/issues"
 Source = "https://github.com/CCRI-POPROX/poprox-recommender"
 
+[tool.hatch.build.targets.wheel.force-include]
+"src/models" = "models/"
```

Comment:
The model artifacts are baked into the `whl` that gets installed into the Docker container for deployment